components/PageSection.tsx

```diff
@@ -20,6 +35,11 @@ export default function PageSection({ title, image, children }: Props) {
             height={210}
             className="h-48 w-48"
           />
+          <div className="flex space-x-3 pt-6">
```

Comment:
I'm not sure if we want to hide the social icons on mobile, which they currently are (see `flex hidden` above, line 30).